dockstore-webservice/src/main/java/io/dockstore/webservice/core/File.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2015 Collaboratory
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+package io.dockstore.webservice.core;
+
+import io.swagger.annotations.ApiModel;
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+/**
+ *
+ * @author xliu
+ */
+@ApiModel(value = "File")
+@Entity
+@Table(name = "file")
+public class File {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private long id;
+
+    @Column
```

Comment:
Use an enum here, not a plain string. 
Use `@Enumerated(EnumType.STRING)` to save it as a string in the database, but we want this locked down on the Java side.